### PR TITLE
lib upgrade

### PR DESCRIPTION
**Pull request by priyankadhiman3006**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ object AppDependencies {
     "ai.x"                    %% "play-json-extensions"              % "0.42.0",
     "org.typelevel"           %% "cats-core"                         % "2.6.1",
     "com.github.kxbmap"       %% "configs"                           % "0.6.1",
-    "uk.gov.hmrc.objectstore" %% s"object-store-client$playVersion"  % "0.37.0",
+    "uk.gov.hmrc.objectstore" %% s"object-store-client$playVersion"  % "0.39.0",
```
Comment (AndyHWChung) [inline project/AppDependencies.scala:19]: this isn't about this dependency, but we could probably update scalmock from 5.1.0 to 5.2.0 below

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@ object AppDependencies {
     "ai.x"                    %% "play-json-extensions"              % "0.42.0",
     "org.typelevel"           %% "cats-core"                         % "2.6.1",
```
Comment (AndyHWChung) [inline project/AppDependencies.scala:17]: is cats 2.7.0 possible here?

File: project/plugins.sbt [hunk 3]
```diff
@@ -5,7 +5,7 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 resolvers += Resolver.typesafeRepo("releases")
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 
-addSbtPlugin("uk.gov.hmrc"               % "sbt-auto-build"     % "3.5.0")
+addSbtPlugin("uk.gov.hmrc"               % "sbt-auto-build"     % "3.6.0")
```
Comment (AndyHWChung) [inline project/plugins.sbt:8]: is 1.9.0 -> 1.9.3 possible for scoverage?

File: project/AppDependencies.scala (hunk 2 above)
Comment (priyankadhiman3006) [reply project/AppDependencies.scala:17]: yes, upgrading in others as well

**Review by AndyHWChung — APPROVED** [review]

File: project/AppDependencies.scala (hunk 2 above)
Comment (priyankadhiman3006) [reply project/AppDependencies.scala]: done, could you please resolve this

Comment (AndyHWChung) [reply project/AppDependencies.scala]: whoops, sorry! :see_no_evil:

**Pull request merged**